zcash_primitives/src/note_encryption.rs

```diff
@@ -231,15 +226,16 @@ fn prf_ock(
 /// let ovk = OutgoingViewingKey([0; 32]);
 ///
 /// let value = 1000;
-/// let rcv = Fs::random(&mut rng);
+/// let rcm = Fs::random(&mut rng);
 /// let cv = ValueCommitment::<Bls12> {
 ///     value,
-///     randomness: rcv.clone(),
+///     randomness: rcm.clone(),
```

Comment:
This change is wrong. We need to generate both `rcv` and `rcm` randomly, and then pass `rcv` here and `rcm` into `create_note`.

Reply:
Also say that the example code is only covering the pre-Canopy case (or generalize it to cover both).